lib/smart_answer_flows/locales/en/state-pension-through-partner.yml

```diff
@@ -384,9 +383,7 @@ en-GB:
           - your own basic State Pension is less than %{higher_basic_state_pension_rate} a week
           - your divorced spouse or civil partner had enough National Insurance contributions
 
-          You’ll get the increase when either you or your former partner reach [State Pension age](/calculate-state-pension), whichever is later.
-
-          ^You won’t get the increase if you remarry or form a new civil partnership before you reach State Pension age.^
+          ^You won’t get the increase if you remarry or form a new civil partnership before you reach [State Pension age.](/calculate-state-pension)^
```

Comment:
Minor: I think it's more common to exclude full stops from links, so:
```
...you reach [State Pension age](/calculate-state-pension).^
```
instead of
```
...you reach [State Pension age.](/calculate-state-pension)^
```